ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -67,29 +67,43 @@ StatusSectionLayout {
             anchors {
                 top: parent.top
                 bottom: footer.top
-                topMargin: 16
-                bottomMargin: 16
+                bottomMargin: 12
+                topMargin: Style.current.smallPadding
                 horizontalCenter: parent.horizontalCenter
             }
-            width: parent.width - 16
-            spacing: 16
+            width: parent.width
+            spacing: 32
             clip: true
 
-            StatusNavigationPanelHeadline {
-                Layout.alignment: Qt.AlignHCenter
-                text: qsTr("Settings")
+            StatusChatInfoButton {
+                id: communityHeader
+
+                title: community.name
+                subTitle: qsTr("%n Member(s)", "", community.members.count)
```

Comment:
Not sure "Members" should be capitalized here

Reply:
You are right! I copied the same behavior from `CommunityColumnView` but in figma it's not capital letter so I'll change both cases, in settings and main community view.